src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAO.java

```diff
@@ -6,27 +6,19 @@
 
 import java.io.IOException;
 import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
 
 @Repository
 @Profile("dev")
-public class MediaEntryDAO extends BaseDAO implements IMediaEntryDAO {
+public class MediaEntryDAO implements IMediaEntryDAO {
```

Comment:
I refactored MediaEntry to use Spring and CrudRepository

Reply:
This removes the inheritance from BaseDAO.  Inheritance is a core concept of OOD, and SOLID principles... so this change may be veering away from good OOD.